easybuild/easyconfigs/p/Perl/Perl-5.30.2-GCCcore-9.3.0.eb

```diff
@@ -21,7 +21,7 @@ dependencies = [
     ('expat', '2.2.9'),  # for XML::Parser
     ('ncurses', '6.2'),  # for Term::ReadLine::Gnu
     ('libreadline', '8.0'),  # for Term::ReadLine::Gnu
-    ('DB', '18.1.32'),  # for DB_File
+    ('DB', '18.1.40'),  # for DB_File
```

Comment:
in-place updates like this are pretty painful...

Another solution to the source tarball for DB 18.1.32 no longer being available is to upload it at https://sources.easybuild.io, which is automatically considered as a fallback URL by EasyBuild.
That's only possible if the software license allows for redistribution though...

Reply:
I don't have a copy of that source file myself. Maybe someone else does (as you say, depending on the license).

Reply:
DB 18.1.x is released under AGPL (+ UPL for examples) license, so sources can be redistributed -> see https://sources.easybuild.io/d/DB/

@dithwick With that, I think we can close this PR as well as #17318?

Reply:
Yep, that's fine with me of course